src/networking/torii.rs

```diff
@@ -48,14 +67,15 @@ impl Torii {
         }
     }
 
-    fn receive(&mut self, command: Command) {
+    fn receive_command(&mut self, payload: &[u8]) {
+        println!("Command payload: {}", String::from_utf8_lossy(payload));
+        let transaction: Transaction = payload.to_vec().into();
         self.pending_tx_cache.add_tx(
-            Transaction::builder(vec![command], "account@domain".to_string())
-                .build()
+            transaction
                 .validate()
                 .expect("Failed to validate transaction."),
         );
-        sleep(Duration::new(2, 0));
+        sleep(Duration::from_millis(1));
```

Comment:
why sleep here?

Reply:
Was used as a simulation of timer - deleted